app/models/order.rb

```diff
@@ -0,0 +1,7 @@
+class Order < ApplicationRecord
+  belongs_to :customer # sets up the association between orders and customer so automatically validates that a customer is present
+
+  validates :product_name, presence: true, format: { with: /\A[a-z\-' ]+\z/i }
+  validates :product_count, presence: true, numericality: { only_integer: true }
+  # validates :customer, presence: true
```

Comment:
I think we may have also commented this out during troubleshooting. Next time, I'll try to help you remember what we adjusted and we can change it back during the session.